doc/modules/ROOT/pages/configuration.adoc

```diff
@@ -80,6 +80,7 @@ SmallRye JWT supports many properties which can be used to customize the token p
 |smallrye.jwt.keystore.verify.key.alias||This property has to be set to identify a public verification key which will be extracted from `KeyStore` from a matching certificate if `mp.jwt.verify.publickey.location` points to a `KeyStore` file.
 |smallrye.jwt.keystore.decrypt.key.alias||This property has to be set to identify a private decryption key if `mp.jwt.decrypt.key.location` points to a `KeyStore` file.
 |smallrye.jwt.keystore.decrypt.key.password||This property may be set if a private decryption key's password in `KeyStore` is different to `smallrye.jwt.keystore.password` when `mp.jwt.decrypt.key.location` points to a `KeyStore` file.
+|smallrye.jwt.keys.remote.startupFetch|false|Set this property to `true` to fetch the keys from the remote location at the application startup.
```

Comment:
Can I suggest to reword this property a little bit ? Something like `smallrye.jwt.resolve-remote-keys-at-startup` ?